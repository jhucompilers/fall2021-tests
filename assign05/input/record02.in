PROGRAM record02;
  TYPE Vec3 = RECORD
    x, y, z: INTEGER;
  END;
  VAR a, b : Vec3;

BEGIN
  READ a.x;
  READ a.y;
  READ a.z;
  READ b.x;
  READ b.y;
  READ b.z;

  WRITE a.x;
  WRITE a.y;
  WRITE a.z;

  WRITE b.x;
  WRITE b.y;
  WRITE b.z;
END.
