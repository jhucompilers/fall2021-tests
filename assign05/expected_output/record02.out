18
44
83
73
80
93
